cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowSupportedProgram.java

```diff
@@ -21,5 +21,5 @@
  */
 
 public enum WorkflowSupportedProgram {
-  MAPREDUCE, SPARK
+  MAPREDUCE, SPARK, CUSTOM_ACTION
```

Comment:
By using configurer, I don't think this and the `Entry` class needs to be in the `cdap-api` module, right?